macros/edr/tests/on_run_end/handle_tests_results.sql

```diff
@@ -308,6 +313,8 @@
     {%- else -%}
         {% set test_name = test_long_name %}
     {%- endif -%}
+    {# Becasue we use compiled sql for fetching data sampling, we currently not using the macro get_model_sql_from_node #}
```

Comment:
I think this comment is not relevant as the macro is called `get_model_sql_from_node` and here it's a test